android/kotlin_guide.md

```diff
@@ -0,0 +1,49 @@
+# Kotlin guide
+This guide is designed to help you get started with [Kotlin], the language we write most of our code in, and to introduce you to some intermediate topics.
+- [Getting Started](#getting-started)
+- [Staying Informed](#staying-informed)
+- [Intermediate Topics](#intermediate-topics)
+
+## Getting Started
+There are multiple ways to get started with Kotlin, depending on how you like to learn. If you like:
+- Reading documentation, the official Kotlin docs are good: start at [Basic Syntax] and work your way through the pages in the side bar
+- Learning while coding, try the [Kotlin Koans] online or in the IDE
+- Courses, try the official [Kotlin for Android Developers][course] course (note: none of our team members have tried this yet)
+- Reading books, try [Kotlin in Action]
+
+Once you've gained some basic familiarity with the language, you can gain more experience by:
+- Watching videos and conference talks on YouTube.
+- [Convert Java to Kotlin with the auto-converter][convert]and change the resulting code into idiomatic Kotlin
+- Doing coding exercises on [exercism.io](https://exercism.io/)
```

Comment:
Neat, I had not heard of this website but it looks cool.

Reply:
https://codewars.com also has a lot of good Kotlin exercises

Reply:
@boek Is there any benefit to codewars over exercism? I'm trying to reduce the number of choices for people to keep it simple.

Reply:
It's worth mentioning all KotlinConf, Google I/O, and Android Dev Summit videos are on Youtube.